### PR TITLE
fix: empty wishlist implementation

### PR DESCRIPTION
**Pull request by AlexanderDevitsky**
## Related Issue

https://vsf.atlassian.net/browse/M2-500

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/modules/wishlist/components/EmptyWishlist.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<template>
```
[the remark this answers is not in the document]
Comment (AlexanderDevitsky) [reply packages/theme/modules/wishlist/components/EmptyWishlist.vue:1]: @bartoszherba yes, I'll add

[the remark this answers is not in the document]
Comment (AlexanderDevitsky) [reply packages/theme/modules/wishlist/components/EmptyWishlist.vue:1]: @bartoszherba done

[the remark this answers is not in the document]
Comment (bartoszherba) [reply packages/theme/modules/wishlist/components/EmptyWishlist.vue:1]: Thank you.

File: packages/theme/modules/wishlist/components/EmptyWishlist.vue [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <SvgImage
+          icon="empty_cart_image"
+          :label="$t('Empty bag')"
+          width="211"
+          height="143"
+          class="empty-wishlist__icon"
+          data-testid="icon"
+        />
+        <SfHeading
+          :title="$t('Your bag is empty')"
```
Comment (bartoszherba) [inline packages/theme/modules/wishlist/components/EmptyWishlist.vue:14]:
```suggestion
          :title="$t('Your wishlist is empty')"
```
```suggestion
          :title="$t('Your bag is empty')"
```

**Review by bartoszherba — APPROVED** [review]

**Pull request merged**